packages/account/src/Components/poa-file-uploader-container/index.js

```diff
@@ -0,0 +1,3 @@
+import PoaFileUploaderContainer from './poa-file-uploader-container';
+
+export default PoaFileUploaderContainer;
```

Comment:
let's move this file and it's respective files to `/poa/file-uploader-container`

Reply:
thanks, done :+1: